### PR TITLE
Audit logging tweaks to support UI:

### PR DESCRIPTION
**Pull request by vadimberezniker**
- Add paging
 - Clear out some redundant fields from requests

**Related issues**: N/A

### CODE REVIEW

**Review by tylerwilliams — APPROVED** [review]
LG w/ nits

File: enterprise/server/auditlog/auditlog.go [hunk 1]
```diff
@@ -192,15 +219,20 @@ func (l *Logger) GetLogs(ctx context.Context, req *alpb.GetAuditLogsRequest) (*a
...
+	qb.AddWhereClause("event_time_usec >= ?", req.GetTimestampAfter().AsTime().UnixMicro())
+	qb.AddWhereClause("event_time_usec <= ?", req.GetTimestampBefore().AsTime().UnixMicro())
+	if req.PageToken != "" {
+		ts, err := strconv.ParseInt(req.PageToken, 10, 64)
+		if err != nil {
+			return nil, err
+		}
+		qb.AddWhereClause("event_time_usec >= ?", ts)
 	}
+	qb.SetLimit(pageSize + 1)
```
[the remark this answers is not in the document]
Comment (tylerwilliams) [reply enterprise/server/auditlog/auditlog.go:231]: ah i see the continue now `if len(resp.Entries) == pageSize {`, ignore this :)

(and thanks for explaining)

File: enterprise/server/auditlog/auditlog.go [hunk 2]
```diff
@@ -27,6 +28,10 @@ var (
 	auditLogsEnabled = flag.Bool("app.audit_logs_enabled", false, "Whether to log administrative events to an audit log. Requires OLAP database to be configured.")
 )
 
+const (
+	pageSize = 20
```
[the remark this answers is not in the document]
Comment (vadimberezniker) [reply enterprise/server/auditlog/auditlog.go:33]: done

File: enterprise/server/auditlog/auditlog.go [hunk 3]
```diff
@@ -178,6 +181,30 @@ func (l *Logger) Log(ctx context.Context, resource *alpb.ResourceID, action alpb
 	}
 }
 
+// cleanRequest clears out redundant noise from the requests.
```
[the remark this answers is not in the document]
Comment (vadimberezniker) [reply enterprise/server/auditlog/auditlog.go:185]: done. i'll also get rid of the redundant group_id fields. i sent out a PR to remove references from server code and after that's rolled out we can update the UI to stop populating it.

**Pull request merged**